pydomcfg/domzgr/zgr.py

```diff
@@ -53,9 +53,8 @@ def sigma(self, grd: str) -> DataArray:
         *) W-points are at integer values - 1/2 (between 0.5 and jpk-0.5)
         """
 
-        kindx = DataArray(
-            range(1, self._jpk + 1), coords={"z": range(self._jpk)}, dims="z"
-        ).astype(float)
+        kindx = DataArray(range(self._jpk), coords={"z": range(self._jpk)}, dims="z")
+        kindx = kindx + 1.0  # to deal with python convention
```

Comment:
to be consistent with ln 60: 
`kindx += 1.0 # Fortran indexing`

Reply:
Can't do it because `int += float` is not allowed with DataArrays.
I don't like it either, so many options that I can't decide what would be the best to clarify what's going on:
```python
kindx = DataArray(np.arange(1., self._jpk+1), coords={"z": range(self._jpk)}, dims="z")
# or
kindx = DataArray(range(1, self._jpk+1), coords={"z": range(self._jpk)}, dims="z").astype("float")
# or
kindx = DataArray(range(self._jpk), coords={"z": range(self._jpk)}, dims="z").astype("float")
kindx += 1.
```
Thoughts?

Reply:
My vote is for: